ngrinder-controller/src/test/java/org/ngrinder/script/repository/MockFileEntityRepository.java

```diff
@@ -25,6 +27,10 @@
 
 	private File userRepoDir;
 
+	public MockFileEntityRepository(Config config, UserContext userContext) {
+		super(config, userContext);
+	}
```

Comment:
Use lombok

Reply:
When I try to use Lombok constructor annotations. It shows error that `Lombok needs a default constructor in the base class`.

Most of mocking classes extend its test target class. It looks hard to apply lombok in test classes.